ext/storage_sources/gcp_store/test/test_gcp_connection.cpp

```diff
@@ -31,4 +31,67 @@
 
 #include "gcp_connection.h"
 
-TEST_CASE("Testing class gcpConnection", "gcp-connection") {}
+TEST_CASE("Testing class gcpConnection", "gcp-connection")
```

Comment:
let's revert this test case, since it sounds like we aren't going to be adding unit testing yet until later.